tools/replay/workers.go

```diff
@@ -106,6 +152,11 @@ func NewClient(w *FileWorker, pace bool) *ClientWorker {
 }
 
 func (w *FileWorker) Run(file string, wg *sync.WaitGroup) {
+	w.latencyDigest = tdigest.NewWithCompression(1000)
```

Comment:
Is 1000 good enough?

Reply:
what is the semantics of this parameter ?